README.md

```diff
@@ -335,6 +335,14 @@ Each command can be executed with `invoke <command>`. All commands support the a
   unittest         Run Django unit tests for the plugin.
 ```
 
+`unittest` uses by default the Postgres DB backend, if you want to test Mysql you must use Invoke environment variables:
+
+```bash
+export INVOKE_NAUTOBOT_CIRCUIT_MAINTENANCE_NAUTOBOT_DB_ENGINE=django.db.backends.mysql
+export INVOKE_NAUTOBOT_CIRCUIT_MAINTENANCE_NAUTOBOT_DB_PORT=3306
+export INVOKE_NAUTOBOT_CIRCUIT_MAINTENANCE_NAUTOBOT_DB_USER=root
```

Comment:
Feels a bit weird to me that setting `INVOKE_NAUTOBOT_CIRCUIT_MAINTENANCE_NAUTOBOT_DB_ENGINE` is really just a roundabout way to set `NAUTOBOT_DB_ENGINE` that actually influences the development environment. Why not just instruct the user to update their `creds.env` file directly instead?

Reply:
I did this way to change the behaviour in the CI: https://github.com/nautobot/nautobot-plugin-circuit-maintenance/pull/159/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR135
Using your proposal, I would have to define to `creds.example.env`, one for Postgres and the other for Mysql, and copy one or the other.
I see both as valid, so if you prefer the one you are proposing, I can change it

Reply:
Something that it's not fully covered with the `creds.env` approach is that we also use this variable to define the [docker-compose files to use](https://github.com/nautobot/nautobot-plugin-circuit-maintenance/pull/159/files#diff-2e746c9eacad6e2c2fdefdc6a665b5ce4607ea384ebf97a729044b79acc778d6R30) (apart of the Nautobot config settings), so you still need to export the `invoke` variables, at least the one for `engine`.